test/eblox/data/taxonomies/tags_test.exs

```diff
@@ -16,9 +16,16 @@ defmodule Eblox.Data.Taxonomies.TagsTest do
 
   def post_id(id), do: "#{@content_dir}/#{id}"
 
-  test "taxonomy with tags" do
-    # TODO: Wait until parsing is completed.
-    Process.sleep(100)
+  test "taxonomy with tags", context do
+    _providers_pid =
+      context.providers
+      |> GenServer.call(:which_children)
+      |> Enum.find(&match?({Eblox.Data.Providers, _, :worker, [Siblings]}, &1))
+      |> elem(1)
+
+    Process.send(Eblox.Test.Messenger, {:listener, self()}, [])
+
+    assert_receive :on_ready
```

Comment:
Love it. Should we just make it a part of `Eblox.ContentCase`? In most of scenarios we want to wait until a provider is ready.

Reply:
Yeah, why not, please do.